src/services/new/witness/electionManager.ts

```diff
@@ -297,6 +314,10 @@ export class ElectionManager {
             deposits: [],
             withdrawels: [],
 
+            //List of weights in the election. Calculated by weights API
+            weights: weights,
+            weight_total: Number(weights.reduce((a, b) => a + b, 0).toFixed(1)),
```

Comment:
`Number.toFixed(1)` would make `1.9999` be `1.9`, but that would be a potential security risk since we would expect it to be `2.0`. The following should achieve that with more accuracy. Although, this solution still is not perfect. The best way would probably be making all weights integers so we don't need to worry about floating point accuracy. 
```suggestion
            weight_total: Math.round(10*weights.reduce((a, b) => a + b, 0)) / 10,
```

Reply:
toFixed is intentional. There was some issue resulting in the value to be like 1.30000000001 despite only 0.6, 1 and 1.3 numbers used in the entire system. Not quite sure how that small variation occurred, but we can sync on it later.

Reply:
Floating-point math is weird and not good when accuracy is important. Let's use integers instead.